app/src/main/java/ch/sdp/vibester/fragment/GameSetupFragment.kt

```diff
@@ -0,0 +1,206 @@
+package ch.sdp.vibester.fragment
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.fragment.app.Fragment
+import ch.sdp.vibester.R
+import ch.sdp.vibester.activity.BuzzerSetupActivity
+import ch.sdp.vibester.activity.ChoosePartyRoomActivity
+import ch.sdp.vibester.activity.LyricsBelongGameActivity
+import ch.sdp.vibester.activity.TypingGameActivity
+import ch.sdp.vibester.api.LastfmApiInterface
+import ch.sdp.vibester.api.LastfmMethod
+import ch.sdp.vibester.api.LastfmUri
+import ch.sdp.vibester.helper.BuzzerGameManager
+import ch.sdp.vibester.helper.GameManager
+import ch.sdp.vibester.helper.TypingGameManager
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.google.gson.Gson
+import dagger.hilt.android.AndroidEntryPoint
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+/*
+* Game Setup fragment with a button in the bottom navigation.
+*/
+@AndroidEntryPoint
+class GameSetupFragment : Fragment(), View.OnClickListener, AdapterView.OnItemSelectedListener{
+    var difficulty = R.string.easy.toString()
+    var game = "local_buzzer"
+    var gameSize = R.string.one.toString()
+    lateinit var gameManager: GameManager
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_game_setup, container, false)
+        val ctx = inflater.getContext();
+
+        view.findViewById<Button>(R.id.local_buzzer_game_button).setOnClickListener(this)
+        view.findViewById<Button>(R.id.local_typing_game_button).setOnClickListener(this)
+        view.findViewById<Button>(R.id.local_lyrics_game_button).setOnClickListener(this)
+        view.findViewById<Button>(R.id.online_buzzer_game_button).setOnClickListener(this)
+        view.findViewById<Button>(R.id.kpopButton).setOnClickListener(this)
```

Comment:
```suggestion

        view.findViewById<Button>(R.id.kpopButton).setOnClickListener(this)
```

Reply:
maybe add a line to separate buttons based on their functionality for readability.

Reply:
done!